go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/canonical/lxd
 
-go 1.20
+go 1.21.7
```

Comment:
It's good to see that version triplet are no longer an issue for any of our tools.